{{copier__project_slug}}/bootstrap-cluster/k3s.yaml

```
@@ -24,8 +25,8 @@ tasks:
     requires:
       vars: [ENV]
     cmds:
-      - tofu -chdir=$TFDIR output control_plane_nodes_public_ips | tr -d '"' > {{ '{{.ENV}}' }}/public_ips.txt
-      - tofu -chdir=$TFDIR output control_plane_nodes_private_ips | tr -d '"' > {{ '{{.ENV}}' }}/private_ips.txt
+      - tofu -chdir=$TFDIR output control_plane_nodes_public_ips | tr -d '"' | sed 's/,/\n/g' > {{ '{{.ENV}}' }}/public_ips.txt
```

We can simplify to a single sed command:
```suggestion
      - tofu -chdir=$TFDIR output control_plane_nodes_public_ips | sed 's/"//g; s/,/\n/g' > {{.ENV}}/public_ips.txt
```